src/audio/Audio.js

```diff
@@ -143,7 +143,7 @@ class Audio extends Object3D {
 
 				// ensure _progress does not exceed duration with looped audios
 
-				this._progress = this._progress % ( this.duration || this.buffer.duration );
+				this._progress = this._progress % ( this.duration || ( this.buffer ? this.buffer.duration : Number.MAX_VALUE ) );
```

Comment:
This line of code is only executed if the audio is playing and if the audio has playback control (which is not true for media elements or streams). In other words, the application calls `play()` without a audio buffer which is wrong to begin with.

TBH, I don't feel the suggested fix is much helpful to users since it hides an erroneous situation. A better fix would be to throw an error at this point or even in `play()` when no buffer is set.